core/src/main/scala/kafka/tools/ReplayLogProducer.scala

```diff
@@ -62,24 +63,27 @@ object ReplayLogProducer extends Logging {
 
   class Config(args: Array[String]) {
     val parser = new OptionParser(false)
-    val zkConnectOpt = parser.accepts("zookeeper", "REQUIRED: The connection string for the zookeeper connection in the form host:port. " +
+    val zkConnectOpt = parser.accepts("zookeeper", "The connection string for the zookeeper connection in the form host:port. " +
       "Multiple URLS can be given to allow fail-over.")
       .withRequiredArg
       .describedAs("zookeeper url")
       .ofType(classOf[String])
       .defaultsTo("127.0.0.1:2181")
```

Comment:
Some commands use `localhost:xxxx` and some use `127.0.0.1:xxxx`. Would be great if we use a single format everywhere.